mysqlTransformer.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import json
+import urllib.request
+import StringIO
+import gzip
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark import SparkConf
+from pyspark import SparkFiles
+
+# Takes arguments: Spark master, Cassandra host, location of the CSV file
+
+conf = SparkConf() \
+    .setAppName("PySpark Cassandra Test") \
+    .setMaster(sys.argv[1]) \
+    .set("spark.cassandra.connection.host", sys.argv[2])
+
+sc = CassandraSparkContext(conf=conf)
+
+#data = sc.textFile(sys.argv[3])
+request = urllib.Request(sys.argv[3])
+request.add_header('Accept-encoding', 'gzip')
+response = urllib.urlopen(request)
+buf = StringIO(response.read())
+f = gzip.GzipFile(fileobj=buf)
+data = f.read()
+
+confPath = SparkFiles.get("conf.json")
+with open(confPath) as f:
+    conf = json.load(f)
+
+indexes = {}
+schema = data.first().split(",")
+
+for i in range(len(schema)):
```

Comment:
Is there a way to directly access the data using an associative array, instead of a double for?